examples/readiness-probe-pod

```diff
@@ -0,0 +1,19 @@
+# Copy of pod.yaml without file extension for test
```

Comment:
Recommend just adding testing-manifests to the tarball as recommended by ixdy, moving the stuff already in there into a test specific subdir (testing-manifests/serviceloadbalancer/), and putting this into testing-manifests/kubectl/